html-css-js/.github/workflows/linters.yml

```diff
@@ -0,0 +1,70 @@
+name: Linters
+
+on: pull_request
+
+env:
+  FORCE_COLOR: 1
+
+jobs:
+  lighthouse:
+    name: Lighthouse
+    runs-on: ubuntu-18.04
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-node@v1
+        with:
+          node-version: "12.x"
+      - name: Setup Lighthouse
+        run: npm install -g @lhci/cli@0.7.x
+      - name: Lighthouse Report
+        run: lhci autorun --upload.target=temporary-public-storage --collect.staticDistDir=.
+  webhint:
+    name: Webhint
```

Comment:
- [ ] Your usual linters are passing and that's great 👏. However, you forgot to update the `linters.yml` file in order to include actions Github needs to parse in order to run eslint linter. I can see that it is here in the directory called `html-css-js` but right here, it is not reachable by Github. What I would suggest you do is to replace the `.github` directory you have here in **the html-css-js folder**, with the `.github` folder which is at **the root level of your website**. By doing this, you will have an updated `linters.yml` file which includes eslint rules to check for JavaScript syntax 😺.  After this, you can just remove that `html-css-js` folder as it is not needed anymore 💡

Reply:
Thank you for the review, I've made the changes requested.